### PR TITLE
Support weight only for ONNX

### PR DESCRIPTION
**Pull request by mengniwang95**
## Type of Change

feature

## Description

Support weight only (RTN, AWQ) for ONNX

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

local test

## Dependency Change?

no

### CODE REVIEW

**Review by xin3he — APPROVED** [review]

**Review by wenhuach21 — APPROVED** [review]
I am not familiar with ONNX, you may ask some others to help review the code

File: neural_compressor/model/onnx_model.py [hunk 1]
```diff
@@ -717,6 +722,33 @@ def match_parent(
 
         return None
 
+    def get_absorb_pairs(self, target_optype):
+        """Find a sequence of input edges based on constraints on parent op_type and index.
+
+        Args:
+            node (str): current node name.
```
Comment (yuwenzho) [inline neural_compressor/model/onnx_model.py:729]: The docstring seems to be not correct.

File: neural_compressor/adaptor/ox_utils/weight_only.py [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+
+    Returns:
+        inputs: prepared inputs.
+        so: session options
+    """
+    from importlib.util import find_spec
+    from neural_compressor.adaptor.ox_utils.util import to_numpy
+    
+    so = ort.SessionOptions()
+    if sys.version_info < (3, 10) and find_spec('onnxruntime_extensions'):  # pragma: no cover
```
Comment (yuwenzho) [inline neural_compressor/adaptor/ox_utils/weight_only.py:301]: Just a remind here, `onnxruntime_extensions` is avaliable for python 3.10. I will fix it in #1138 after this PR is merged,

**Review by yuwenzho — APPROVED** [review]

**Pull request merged**